src/src_V2/modules/manager/financial/components/receivables/Receivables.jsx

```diff
@@ -0,0 +1,69 @@
+import { useCallback, useEffect, useState } from 'react';
+
+import { rowsPerPageOptions, receivableColumns } from '../../constants';
+import receivablesList from '../../mock/receivable';
+import { remainingCost } from '../../utils/cost';
+import { Layout } from '../../../../../components';
+import ControlPanel from './control-panel/ControlPanel';
+import ReceivableTable from './table/ReceivableTable';
+
+export default function Receivables() {
+  const [receivables, setReceivables] = useState([]);
+  const [receivablesLoading, setReceivablesLoading] = useState(false);
+  const [filteredReceivables, setFilteredReceivables] = useState([]);
+  const [searchTxt, setSearchTxt] = useState('');
+
+  // getting Receivables from backend
+  function getReceivables() {
+    setReceivablesLoading(true);
+
+    const timeout = setTimeout(() => {
+      processReceivables(receivablesList());
+      setReceivablesLoading(false);
+    }, 500);
+
+    return () => {
+      clearTimeout(timeout);
+    };
+  }
+
+  function processReceivables(receivables) {
+    const processedReceivables = receivables.map((receivable) => {
+      receivable.remaining = remainingCost(receivable.totalAmount, receivable.paidAmount);
+      return receivable;
+    });
+    setReceivables(processedReceivables);
+  }
+
+  const filterReceivables = useCallback(() => {
+    // filtering receivables by Tour Name/ID/Partner
+    const filteredReceivables = receivables.filter(
+      (item) =>
```

Comment:
I think this approach will be more readable: `({ tourName, receivableId, partner }) => [tourName, receivableId, partner].some(item => item.toLowerCase().includes(searchTxt))`